package.json

```diff
@@ -1,19 +1,22 @@
 {
-  "name": "dogs",
+  "name": "@formidablelabs/dogs",
   "version": "1.0.0",
   "description": "Formidable dogs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "author": "@FormidableLabs",
+  "license": "MIT",
   "repository": {
-    "type": "git",
     "url": "git+https://github.com/FormidableLabs/dogs.git"
   },
-  "author": "",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/FormidableLabs/dogs/issues"
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com"
+  },
+  "scripts": {
+    "build": "rimraf dist/ && tsc"
```

Comment:
Do we want to add a `preversion` workflow to make sure that everything appropriate is released? A la https://github.com/FormidableLabs/trace-deps/blob/master/CONTRIBUTING.md#releasing-a-new-version-to-npm

Reply:
What will be published is included in the [GithHub workflow](https://github.com/FormidableLabs/dogs/blob/typescript-package/.github/workflows/gpr-publish.yaml#L22) via `npm publish --dry-run`. 

[Example](https://github.com/FormidableLabs/dogs/runs/1260574891?check_suite_focus=true#step:6:9)